src/main/resources/org/apache/fluo/resources/java-checkstyle.xml

```diff
@@ -30,6 +28,8 @@
   </module>
 
   <module name="TreeWalker">
+    <module name="SuppressWithNearbyCommentFilter"/>
```

Comment:
> I thought that's basically what my log message said, but I can reword it if it's not clear.

Before I looked at the code changes, based on the commit message I was expecting this change to change a pointer to an externally versioned resource.  When I looked at the code changes I was not sure what was what, not sure if a better log messages would have helped.  I don't care if you change the message, just made that comment about the log message based on my first impression.